proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -11,20 +12,23 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/auth/types";
 // for basic account functionality. Any custom account type should extend this
 // type for additional functionality (e.g. vesting).
 message BaseAccount {
+  option (legacy_amino.name)          = "cosmos-sdk/BaseAccount";
```

Comment:
Why calling it `legacy_amino` rather than simply `amino`. The codec name is `amino`, not `legacy_amino`. And the codec will stay.
I think `legacy` prefix is good for some function names, but not for proto level domain names.

Reply:
`amino` is better 👍🏻

Reply:
updated